flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/Execution.java

```diff
@@ -131,6 +133,35 @@
 	
 	private SerializedValue<StateHandle<?>> operatorState;
 
+	/* Lock for updating the accumulators atomically. */
```

Comment:
If you consider comment lines as empty (non-code) lines, then it follows the classes' style.